README.md

```diff
@@ -0,0 +1,95 @@
+# FarmLink
```

Comment:
Blank lines are important in Markdown.  Some of them are technically optional, but they can quickly become mandatory and mess up your layout.  I'm adding suggested changes to add all of the blank lines that will ensure your layout is correct.